prow/jobs.yaml

```diff
@@ -200,6 +200,8 @@ kubernetes/test-infra:
     containers:
     - name: builder
       image: gcr.io/k8s-testimages/test-infra-go-test:0.0
+      command: ["/bin/bash", "-c"]
+      args: ["git clone https://github.com/kubernetes/test-infra && ./test-infra/jenkins/bootstrap.py --repo=k8s.io/kubernetes --pull=${PULL_REFS} --job=pull-test-infra-prow-test"]
```

Comment:
It is, but I'm getting a super low-level docker error and it's one of the few differences between this image and the bazel build image, which is working properly.

I'm overriding it here to cut down on the differences.